### DISCUSSION TITLE
studies are not loading

### DISCUSSION THREAD

**Discussion by Avinash-c-gowda** [nroduit/Weasis#451] [General]
HI @nroduit ; Some system studies are not loading  could u please tell why it is happening. Thanks in advance

An operation related to DICOM image loading took place. The log entry indicates that the process was successful ("INFO").
The type of DICOM object loaded was a DICOMDIR.
The series unique identifier (UID) is "***********."
The modality of the images in the series is "XX".
The number of images (nbImages) in the series is 0.
The size of the loaded data is 31,472,614 bytes.
The loading time was 9328 milliseconds (9.328 seconds).
The loading rate was approximately 3373.99 bytes per second.
No errors occurred during the loading process ("errors:0").
On the same timestamp, the log also records the removal of patient and study information:

Patient Name: "XX" was removed from the DICOM model.
Study Date: "XXX" was removed from the DICOM model.
It seems like a DICOM viewer or explorer software (possibly "org.weasis.dicom.explorer") was used, and it loaded a DICOMDIR but found no actual images in the series. After that, it removed the information related to the patient and study named "XX, MS" and "XXX" respectively.

**Comment by nroduit** [comment]
Without an example, it's hard to answer. We need to determine where the problem lies. Does the exam load by opening the DICOM file directory directly, without using DICOMDIR?

**Reply by Avinash-c-gowda** [reply]
yes,  i tried to load from our own  PACS also,  this is happening in only one system,  By using DICOMDIR also not loading

**Comment by nroduit** [comment]
If there are no errors in the logs, it looks like a classic problem where the DICOMDIR is not correctly constructed.

**Comment by Avinash-c-gowda** [comment]
for windows7 won't load rt; but i tried in windows11 system also

The size of the loaded data is (31,472,614) printing.

**Comment by Avinash-c-gowda** [comment]
could u please  clarify   windows7 and  windows11 weasis viewer is working?
